ibc-clients/ics07-tendermint/cw-contract/src/tests/helper.rs

```diff
@@ -0,0 +1,26 @@
+use std::str::FromStr;
+
+use cosmwasm_std::testing::mock_info;
+use cosmwasm_std::{coins, MessageInfo};
+use ibc_client_tendermint::types::ConsensusState;
+use ibc_core::primitives::Timestamp;
+use tendermint::Hash;
+
+pub fn dummy_msg_info() -> MessageInfo {
+    mock_info("creator", &coins(1000, "ibc"))
+}
+
+pub fn dummy_checksum() -> Vec<u8> {
+    hex::decode("2469f43c3ca20d476442bd3d98cbd97a180776ab37332aa7b02cae5a620acfc6")
+        .expect("Never fails")
+}
+
+pub fn dummy_sov_consensus_state(timestamp: Timestamp) -> ConsensusState {
+    ConsensusState::new(
+        vec![0].into(),
```

Comment:
This should work too.

```suggestion
        &[0].into(),
```

Reply:
[6de21ed](https://github.com/cosmos/ibc-rs/pull/1167/commits/6de21edb62137b9d74daf1bcacf3b626e45013cc)